Terraform/GCP/cluster/startup-master.sh

```diff
@@ -10,20 +10,66 @@ set -uo pipefail
 export INSTALL_K3S_VERSION=${k3s_version}
 export K3S_DATASTORE_ENDPOINT="mysql://${db_username}:${db_password}@tcp(${db_host}:3306)/k3s"
 export K3S_TOKEN=${token}
+file="/var/lib/rancher/k3s/server/tls/client-admin.crt"
 
-curl -sfL https://get.k3s.io | sh -s - server \
-  --disable-cloud-controller \
-  --write-kubeconfig-mode 644 \
-  --kubelet-arg="cloud-provider=external" \
-  --tls-san "${external_lb_ip_address}" \
-  --tls-san "${external_hostname}" \
-  --tls-san "${internal_lb_ip_address}" \
-  --disable traefik \
-  --disable servicelb \
-  --disable local-storage
+# Function generate custom ca-certs
+generate_custom_certs() {
+  # Create folder for certs
+  mkdir -p /var/lib/rancher/k3s/server/tls
 
-sleep 10
+  # Save root and intermediate ca-certs
+  echo "${ROOT_CA_PEM_CERT}" > /var/lib/rancher/k3s/server/tls/root-ca.pem
+  echo "${INTERMEDIATE_CA_PEM}" > /var/lib/rancher/k3s/server/tls/intermediate-ca.pem
+  echo "${INTERMEDIATE_CA_KEY}" > /var/lib/rancher/k3s/server/tls/intermediate-ca.key
 
+  # Generating custom certs
+  curl -sL https://github.com/k3s-io/k3s/raw/master/contrib/util/generate-custom-ca-certs.sh > generate-custom-ca-certs.sh
+  chmod +x generate-custom-ca-certs.sh
+  ./generate-custom-ca-certs.sh
+  ls /var/lib/rancher/k3s/server/tls
```

Comment:
maybe this ls is not needed in the final version?:)

Reply:
It will generate the certificates. Without this, k3s will not be installed.